src/App.js

```diff
@@ -36,10 +40,27 @@ const App = () => {
         </ShoppingCart>
       </Header>
       <Content>
-      {currentPage === "Home" ? <Home /> : <ProductList products={productsData.results} categories={productCategoriesData.results} />}
-        <div>
+        <Switch>
+          <Route exact path='/home'>
```

Comment:
It is a good practice to create a component just for the router

Reply:
A better way to define routes or at least one that o find easier to read and maintain  is the following
```suggestion
          <Route exact path='/home' component={Home} />
          <Route exact path='/products' component={ProductList}>
```